config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+  patch "/movies/:id", to: "movies#update"
```

Comment:
As the other comments say, I think this could be more accurately represented as a create action. But since we're not really creating a Movie but a relationship _between_ a movie and an actor, we could use a different controller or path. I suggest experimenting with this route and see how it could work with the form, too: 

```
post "/movie_actors", to: "movie_actors#create"
```